docs/book/src/cronjob-tutorial/testdata/project/.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Kubebuilder DevContainer",
-  "image": "docker.io/golang:1.24",
+  "image": "golang:1.24",
```

Comment:
Hi @dongjiang1989

I am pinging you to make you aware.
It was my mistake since I did not review your PR.